sdk/core/azure-core/src/main/java/com/azure/core/http/HttpPipelineBuilder.java

```diff
@@ -41,7 +41,12 @@
     private List<HttpPipelinePolicy> pipelinePolicies;
 
 
-    HttpPipelineBuilder() {
+    /**
+     -     * Creates a builder that can configure options for the HttpPipeline before creating an instance of it.
+     -     *
+     -     * @return A new {@link HttpPipelineBuilder} to create a HttpPipeline from.
+     */
+    public HttpPipelineBuilder() {
```

Comment:
Can you please fix up the javadoc here. Thanks.

Reply:
on it